project/src/main/java/com/google/sps/servlets/formHandlerServlet.java

```diff
@@ -0,0 +1,36 @@
+package com.google.sps.servlets;
+
+import java.io.IOException;
+import javax.servlet.annotation.WebServlet;
+import javax.servlet.http.HttpServlet;
+import javax.servlet.http.HttpServletRequest;
+import javax.servlet.http.HttpServletResponse;
+
+@WebServlet("/form-handler")
+public class formHandlerServlet extends HttpServlet {
+
+    @Override
+    public void doPost(HttpServletRequest request, HttpServletResponse response) throws IOException {
+
+        // Get the value entered in the form.
+        String textValue = request.getParameter("signupUsername");
+        String textValue2 = request.getParameter("signupEmail");
+        String textValue3 = request.getParameter("signupPassword");
+        String textValue4 = request.getParameter("confirmSignupPassword");
+
+        // Print the value so you can see it in the server logs.
+        System.out.println("You submitted: " + textValue);
```

Comment:
Optional: Lines 22 - 25 can be condensed into a single print statement.

Reply:
will do!